tests/OpenCFP/Http/Controller/Admin/TalksControllerTests.php

```diff
@@ -0,0 +1,184 @@
+<?php
+namespace OpenCFP\Tests\Http\Controller\Admin;
+
+use Mockery as m;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\HttpFoundation\Session\Storage\MockFileSessionStorage;
+use OpenCFP\Application;
+use OpenCFP\Environment;
+
+class AdminTalkControllerTest extends \PHPUnit_Framework_TestCase
+{
+    public $app;
+
+    public function setup()
+    {
+        // Create our Application object
+        $this->app = new Application(BASE_PATH, Environment::testing());
+
+        // Create a test double for our User entity
+        $user = m::mock('OpenCFP\Domain\Entity\User');
```

Comment:
Gonna leave it for now. This is how it's done throughout the codebase so it'd make more sense to make an effort across the codebase to do this if we want to. I also want to make sure there's no sketchy reason we have a 5.4 allowance in composer but aren't testing for it. I'll create an issue to do so after we figure out why there's a difference. Pretty sure that's an oversight, but want to double check.